charts/access/slack/README.md

```diff
@@ -0,0 +1,182 @@
+# Teleport Access Request Slack Plugin
+
+This chart sets up and configures a Deployment for the Access Request Slack plugin.
+
+## Installation
+
+### Prerequisites
+
+First, you'll need to create a Teleport user and role for the plugin. The following file contains a minimal user that's needed for the plugin to work:
+
+```yaml
+---
+kind: role
+version: v5
+metadata:
+  name: teleport-plugin-slack
+spec:
+  allow:
+    logins:
+    - teleport-plugin-slack
+    rules:
+    - resources:
+      - access_request
+      verbs:
+      - list
+      - read
+      - update
+  options:
+    forward_agent: false
+    max_session_ttl: 8760h0m0s
+    port_forwarding: false
+---
+kind: user
+version: v2
+metadata:
+  name: teleport-plugin-slack
+spec:
+  roles:
+    - teleport-plugin-slack
+```
+
+You can either create the user and the roles by putting the YAML above into a file and issuing the following command  (you must be logged in with `tsh`):
+
+```
+tctl create user.yaml
+```
+
+or by navigating to the Teleport Web UI under `https://<yourserver>/web/users` and `https://<yourserver>/web/roles` respectively. You'll also need to create a password for the user by either clicking `Options/Reset password...` under `https://<yourserver>/web/users` on the UI or issuing `tctl users reset teleport-plugin-slack` in the command line.
+
+The next step is to create an identity file, which contains a private/public key pair and a certificate that'll identify us as the user above. To do this, log in with the newly created credentials and issue a new certificate (525600 and 8760 are both roughly a year in minutes and hours respectively):
+
+```
+tsh login --proxy=access-dev.teleportinfra.dev --auth local --user teleport-plugin-slack --ttl 525600
+```
+
+```
+tctl auth sign --user teleport-plugin-slack --ttl 8760h --out teleport-plugin-slack-identity
+```
+
+Alternatively, you can execute the command above on one of the `auth` instances/pods.
+
+The last step is to create the secret. The following command will create a Kubernetes secret with the name `teleport-plugin-slack-identity` with the key `auth_id` in it holding the contents of the file `teleport-plugin-slack-identity`:
+
+```
+kubectl create secret generic teleport-plugin-slack-identity --from-file=auth_id=teleport-plugin-slack-identity
+```
+
+### Installing the plugin
+
+```
+helm repo add teleport https://charts.teleport.sh/
```

Comment:
This is a different address to our regular Helm repo (https://charts.releases.teleport.dev) - is that intentional?

Reply:
It's definitely a mistake. Thanks for catching it!